src/main/java/core/basesyntax/Counter.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Counter {
+    private static final int NUMBER = 100;
+    private int value;
+
+    public void countUp() {
+        value++;
+    }
+
+    public int getValue() {
+        return value;
+    }
+
+    public boolean isFinish() {
+        return value != NUMBER;
+    }
```

Comment:
this method creates some logic that should better be present in our threads' `run()` method than in the counter class